### PR TITLE
Updated the prompts

### PR DESCRIPTION
**Pull request by RahAmudha**
Changed the true or false prompt to indicate whether the question is true or false, and made the other prompts more specific.

### CODE REVIEW

File: backend/app/services/service.py [hunk 1]
```diff
@@ -60,9 +60,8 @@ def service_tf(data):
     
     #! prompt engineered goes here
     prompt = """
-You are a bot that generates practice problems based on an academic keyword, difficulty, and the example that the user gives.
-The difficulty can be a number between 1 and 10, with 1 being the easiest and 10 being the hardest.
-Generate 10  true and false problems with answers about %s at difficulty level %s and based on this example: %s. In JSON format.
+Generate 10 true and false problems with answers about %s at difficulty level %s and based on this example: %s. Indicate that this is a True or False question
+by writing 'TRUE or FALSE: ' at the beginning of the question. The answers should be either 'true' or 'false' in lower case.
```
Comment (nareshkanduru) [inline backend/app/services/service.py:64]: In Json Format?

File: backend/app/services/service.py [hunk 2]
```diff
@@ -82,8 +81,7 @@ def service_mc(data):
 Generate 10 Multiple Choice questions and answers with options about %s at difficulty level %s and based on this example: %s. 
 With 'questions' being the name of the array, 
 'question' being a key for every question,
-3 choices labeled A B or C with 'choice' being an key value pair where the value is an array with the choices
-and 'answers' being the key for right option
+'choices' being an array of strings that start with the letter choices, 'A', 'B', or 'C' and end with their respective answers.
```
Comment (nareshkanduru) [inline backend/app/services/service.py:84]: Json Format?

**Review by nareshkanduru — CHANGES_REQUESTED** [review]
@RahAmudha should the prompts asking to return in json format?

File: backend/app/services/service.py (hunk 1 above)
Comment (RahAmudha) [reply backend/app/services/service.py:64]: the first prompt at the beginning already specifies to return as a JSON so theres no need to repeat it

**Review by billzhang7d3 — APPROVED** [review]
reminder to change the test scripts as a result

**Pull request merged**